easybuild/easyconfigs/p/Python/Python-2.7.8-intel-2015a.eb

```diff
@@ -0,0 +1,99 @@
+name = 'Python'
+version = '2.7.8'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.9.1'
+scipyversion = '0.14.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# libffi build in python is still broken, see http://bugs.python.org/issue4130
+patches = ['python-2.7_libffi-include-xmmintrin.patch']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+#   ('OpenSSL', '1.0.1g'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+#   nice to have an up to date openssl for security reasons
+]
+
+# order is important!
+# Version updated 08/JULY/2014
+exts_list = [
+    ('setuptools', '5.4.1', {
+        'source_urls': ['http://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('pip', '1.5.6', {
+        'source_urls': ['http://pypi.python.org/packages/source/p/pip/'],
+    }),
+    ('nose', '1.3.3', {
+        'source_urls': ['http://pypi.python.org/packages/source/n/nose/'],
+    }),
+    ('numpy', numpyversion, {
+        'source_urls': [('http://sourceforge.net/projects/numpy/files/NumPy/%s' % numpyversion, 'download')],
+        'patches': [
+            'numpy-1.8.0-mkl.patch', # % numpyversion,
+        ],
+    }),
+    ('scipy', scipyversion, {
+        'source_urls': [('http://sourceforge.net/projects/scipy/files/scipy/%s' % scipyversion, 'download')],
+    }),
+    ('blist', '1.3.6', {
+        'source_urls': ['http://pypi.python.org/packages/source/b/blist/'],
+    }),
+    ('mpi4py', '1.3.1', {
+        'source_urls': ['https://mpi4py.googlecode.com/files/'],
+    }),
+    ('paycheck', '1.0.2', {
+        'source_urls': ['http://pypi.python.org/packages/source/p/paycheck/'],
+    }),
+    ('argparse', '1.2.1', {
+        'source_urls': ['http://argparse.googlecode.com/files/'],
+    }),
+    ('lockfile', '0.9.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/l/lockfile/'],
+    }),
+    ('Cython', '0.20.2', {
+        'source_urls': ['http://www.cython.org/release/'],
+    }),
+    ('six', '1.9.0', {
```

Comment:
hmm, we should probably use six v1.7.3 here, which was available at the time the other versions listed here were picked?